metro.config.js

```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  resolver: {
-    sourceExts: ['jsx', 'js']
-  }
+
```

Comment:
Why is this file empty..?

Reply:
the metro configuration was causing conflicts with the vector icons in the simulator render, so we had to remove them, but for safety's sake we left the file as there could be a little node module somewhere that depends on its existence, even if it's empty